ngx-fudis/package.json

```diff
@@ -30,6 +30,7 @@
     "@angular/platform-browser-dynamic": "^18.2.13",
     "@angular/router": "^18.2.13",
     "eslint-config-prettier": "^10.0.1",
+    "remark-gfm": "^4.0.1",
```

Comment:
Isn't this plugin specifically meant for **markdown table support** and since we are not using any, why do we need it?

Reply:
We are using tables in our mdx files. This plugging ads support for markdown and <table> -tags

Reply:
Seemed to work without, maybe some update issue? Removed Markdown tag from InformationSecurityManagement doc page